.github/workflows/code_review.yml

```diff
@@ -2,32 +2,36 @@ name: Code Review
 
 on:
```

Comment:
The PR diff extraction step has been updated to use `git diff origin/main HEAD` instead of installing Python packages and running a script. This change simplifies the workflow and eliminates the need for Python dependencies.